vault_contract/src/contract.rs

```diff
@@ -67,3 +124,18 @@ fn validate_did(e: &Env, did: &String) {
         panic_with_error!(e, ContractError::DidRevoked)
     }
 }
+
+fn validate_issuer(e: &Env, issuer: &Address, did: &String) {
+    issuer.require_auth();
+
+    let issuers: Map<Address, Issuer> = storage::read_issuers(e, did);
+    if issuers.is_empty() {
+        panic_with_error!(e, ContractError::DidWithoutIssuers)
+    }
```

Comment:
@MarioRodriguezS @juanmagiraldor Do you guys consider this ContractError is necessary? maybe by returning `IssuerNotFound` is enough... Let me know what you think!